src/event.rs

```diff
@@ -281,6 +282,7 @@ impl<N: Notify> Processor<N> {
             last_modifiers: Default::default(),
             pending_events: Vec::with_capacity(4),
             window_changes: Default::default(),
+            selection_buffer: ::copypasta::Buffer::Primary,
```

Comment:
Using an absolute path inside the code like this is usually not optimal. Since the buffer is used multiple times in this file, I propose importing it at the beginning of it. If the concern was that `Buffer` doesn't really say much about what it actually is (which would be understandable), I'd propose just importing it as `ClipboardBuffer`.